models/cycle_gan_semantic_model.py

```diff
@@ -98,7 +100,10 @@ def __init__(self, opt):
             self.criterionGAN = loss.GANLoss(opt.gan_mode).to(self.device)
             self.criterionCycle = torch.nn.L1Loss()
             self.criterionIdt = torch.nn.L1Loss()
-            self.criterionCLS = torch.nn.modules.CrossEntropyLoss()
+            if not opt.regression:
+                self.criterionCLS = torch.nn.modules.CrossEntropyLoss()
+            else:
+                self.criterionCLS = torch.nn.modules.MSELoss()
```

Comment:
Let's have an option for L1 loss as well

Reply:
ok!